redaxo/src/core/lib/request.php

```diff
@@ -123,6 +123,20 @@ public static function unsetSession($varname)
         unset($_SESSION[rex::getProperty('instname')][$varname]);
     }
 
+    /**
+     * clear all redaxo session contents (the session itself stays alive)
+     *
+     * @throws rex_exception
+     */
+    public static function clearSession()
+    {
+        if (PHP_SESSION_ACTIVE != session_status()) {
+            throw new rex_exception('Session not started, call rex_login::startSession() before!');
+        }
+
+        unset($_SESSION[rex::getProperty('instname')]);
```

Comment:
Damit wird auch die Frontendsession von z.B. YCom gelöscht.. Am Liebsten wäre es mir, wenn nur $_SESSION[rex::getProperty('instname')]['backend'] gelöscht würde.

Reply:
Siehe https://github.com/redaxo/redaxo/issues/2436

Es ist gewünscht dass werte die via rex_set_session() gesetzt werden hier beim login gecleared werden. Dort gibt es aktuell keine Unterscheidung zwischen frontend und backend.

Reply:
@staabm ich werde dann in der ycom ausgeloggt mit der aktuellen variante